config/config.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+const _ = require("lodash");
+const fs = require("fs");
+const path = require("path");
+
+const envConfig = process.env.NODE_ENV === "DEVTOOLS_PANEL" ?
+  require("./firefox-panel.json") : require("./development.json");
+
+const localConfig = fs.existsSync(path.join(__dirname, "./local.json")) ?
+  require("./local.json") : {};
+
+let config = _.merge({}, envConfig, localConfig);
+const originalConfig = Object.assign({}, config);
+
+// only used for testing purposes
+function stubConfig(stub) {
+  config = stub;
+}
+
+function getConfig() {
+  return config;
+}
+
+function resetConfig(stub) {
```

Comment:
I prefer the other API with just `getConfig` and `setConfig`, with the latter returning the previous config. It's simpler and more functional (no need to worry about originalConfig in here).